src/styles/utils/colorFilterStyle.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+import "@testing-library/jest-dom";
+
+import { colorFilterStyle } from "./colorFilterStyle";
+
+import renderWithTheme from "@/test-helpers/renderWithTheme";
+
+describe("colorFilterStyle", () => {
+  it("should correctly handle prop 'filter' set to OakColorFilterToken", () => {
+    const StyledComponent = styled.div`
+      ${colorFilterStyle}
+    `;
+    const { getByTestId } = renderWithTheme(
+      React.createElement(StyledComponent, {
```

Comment:
This is a very strange way to instantiate a component. How about 
`<StyledComponent data-test-id="test" $colorFilter="red"/>`